### DISCUSSION TITLE
周杰倫演唱會增加了部分區域售票

### DISCUSSION THREAD

**Discussion by Jinyuu17** [1101-GS4719-C/Python-Community-GS4719-C#2240] [General]
上禮拜三大家是不是跟我一樣想搶周杰倫大巨蛋演唱會的門票，但都沒搶到呢。現在官網釋出有視線遮蔽區域的座位了，這禮拜六(11/02)一樣在拓元搶票。上次還在糾結要不要買黃牛票，畢竟糾結倫是我的偶像，但被查到什麼都沒有。

大家準備好要再參加一次手速比賽的敗部復活賽嗎，建議大家可以多練練驗證碼的輸入，並且要先登入會員。我是會用python 的 selenium 來輔助我搶票，並用已經訓練好的機器人判讀驗證碼。在此，住大家都可以進場看周杰倫的演唱會。

**Comment by Boying12** [comment]
上周在周杰倫的演唱會門票開賣時，我真的很期待能夠搶到票，然而卻因為人潮洶湧、網站崩潰等種種原因，最終未能如願。心中不免感到失落，畢竟周杰倫的音樂陪伴了我許多年，他的演唱會對我來說是一次難得的音樂盛宴。每當聽到他的歌曲，心中都充滿了回憶與感動，能親自到現場感受那份氛圍，實在是每個粉絲的夢想。
